indico/MaKaC/webinterface/rh/conferenceModif.py

```diff
@@ -1512,7 +1512,7 @@ def _process(self):
         if not attachments:
             flash(_('The selected contributions do not have any materials.'), 'warning')
             return redirect(url_for('event_mgmt.confModifContribList', self._conf))
-        return self._generate_zip_file(attachments)
+        return self._generate_zip_file(files_holder=attachments, name_suffix=self.event_new.id)
```

Comment:
files_holder is the first argument and required so i'd pass it as a posarg, not a kwarg